### DISCUSSION TITLE
模板能否备份或导出？

### DISCUSSION THREAD

**Discussion by Aren-py** [windingwind/zotero-better-notes#395] [Q&A] [answered]
应用场景：将A电脑中的模板备份或导出，然后导入新的B电脑。目前能备份的插件如tara，无法将bn的模板进行备份，不知道可有其他办法？

**Comment by dawnlh** [comment]
同问，没找到模板文件保存的位置在哪

**Comment by windingwind** [comment]
目前没有导出或保存功能。后续会加导出功能

**Comment by windingwind** [comment, marked as answer]
v1.1.4-2 or higher supports template backup/restore and template share.
